### ISSUE TITLE
E1 Heating failed. Marlin 2.0

### ISSUE THREAD

**Issue by blueeagle69** [MarlinFirmware/Marlin#14472]
Since moving to my SKR 1.3 and Marlin 2.0, i'm getting a lot of E1 Heating Failed messages, with only E0 enabled.

It normally happens when the the printer is heating and I home the printer.
I've tried with #define PROBING_HEATERS_OFF enabled and disabled, but I'm still getting it.
I've attached my configuration files.
Hopefully this will help.
Thank you.
[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/3347537/Marlin.zip)

**Comment by Halestorm42** [comment]
I'm having a similar problem. I am running a SKR 1.3 and Marlin 2.0 on an Ender 3.
When I try to heat for a print with a target temp of 200 it waffles between ~188 and ~195 a couple times, then gives E1 heating failed. I only have one extruder enabled.
Config files also attached.
Thanks!
[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/3352739/Marlin.zip)

**Comment by thinkyhead** [comment]
I will have an SKR 1.3 for testing soon, so I can test the thermistor readings at that time.

**Comment by Halestorm42** [comment]
I was actually able to solve my issue in the past couple hours. The error saying E1 heating failed is an unfixed text error according to [#12625](https://github.com/MarlinFirmware/Marlin/issues/12625) and should say E0.

As for the heating issue (at least my heating issue) I found [this](https://3dprinting.stackexchange.com/questions/7720/hotend-does-not-reach-set-temperature) post on stackexchange recommending calibrating PID values. I found instructions [here](https://reprap.org/wiki/PID_Tuning) and entered the resulting values into my firmware and the problem disappeared.
